app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,9 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.USE_FINGERPRINT"/>
```

Comment:
Does the USE_FINGERPRINT require the user to agree to a permission before use?

Reply:
I am reading that its not marked dangerous so it doesn't need user permission, is that correct?

Reply:
I tested this and no permission request required.

Reply:
Sorry for the delay of my response. As you tested, there is no need to ask the user for this permission.